pages/login.js

```diff
@@ -14,8 +14,8 @@ const Login = () => {
     <Background imagePath="/background.png">
       <div className="h-full w-80 m-auto flex flex-col justify-center items-center">
         <div className="mb-8">
-          <div className="mb-5">
-            <Image src={logo} alt="logo" height={48} />
+          <div className="h-10 mb-6">
+            <Image src={logo} alt="logo" layout="fixed" width="327" height="40" />
```

Comment:
```suggestion
            <NimbleLogo />
```

What about having a dedicated component so that we do not need to duplicate the imports/size/alt/layout... :) 
(then we can reuse this component in the login page as well)

Reply:
Note: As this  logo is also present in the top corner of other screens (with different margins, I would recommend to **not** include the parent div into this component :) )

Reply:
actually, we have two different logos across the application - added the other one in 62702641a4f14b18d1a77bfdc6c11f87694fc841
After the trick you shared on https://github.com/tyrro/react-survey/pull/52/files#r838202648, we don't need the size/layout props too, therefore removed those and didn't DRY this one